src/styles/bloks/_card.scss

```diff
@@ -0,0 +1,107 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  max-width: ($card-with + 48px) * 4;
+  box-sizing: content-box;
+
+  &__card {
+    width: $card-with;
+    margin: 0 48px 46px 0;
+    padding: 32px 16px 16px;
+  }
+}
+
+.card {
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  color: $main-color;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__image {
+    display: flex;
+    width: 100%;
+    height: 134px;
+    text-align: center;
+    align-self: center;
+    padding: 0 3px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    text-align: left;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: #616070;
+  }
+
+  &__stars {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    &-reviews {
+      position: relative;
+      font-weight: 400;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    line-height: 18px;
+
+    &-title {
+      color: #616070;
+      font-weight: 400;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__buy-button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    padding: 11px 0;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #fff;
+    background: $link-color;
+    outline-color: $main-color;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    transition: 0.5s ease-in-out;
```

Comment:
don't use it like that, because it's the same as transition: all 0.5s or transition-duration: 0.5s;
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/222706513-a46dc46c-e792-43dd-81b6-cd95da58572e.png)

```suggestion
    transition: color $transitionDuration ease-in-out, ....;
```